src/dawn/engine_dawn.h

```diff
@@ -68,15 +68,27 @@ class EngineDawn : public Engine {
       const ::amber::PipelineCommand* command) {
     return pipeline_map_[command->GetPipeline()].render_pipeline.get();
   }
-
+  // Returns the Dawn-specific compute pipeline for the given command,
+  // if it exists.  Returns nullptr otherwise.
+  ComputePipelineInfo* GetComputePipeline(
+      const ::amber::PipelineCommand* command) {
+    return pipeline_map_[command->GetPipeline()].compute_pipeline.get();
+  }
   // Creates and attaches index, vertex, storage, uniform and depth-stencil
   // buffers. Sets up bindings. Also creates textures and texture views if not
-  // created yet.
+  // created yet. Used in the Graphics pipeline creation.
   Result AttachBuffersAndTextures(RenderPipelineInfo* render_pipeline);
+  // Creates and attaches index, vertex, storage, uniform and depth-stencil
+  // buffers.Used in the Compute pipeline creation.
```

Comment:
Nit: extra space after the period and before "Used..."

No need to block this MR